spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySource.java

```diff
@@ -108,10 +112,7 @@ public Object getProperty(String name) {
     FeatureSet initProperties(FeatureSet featureSet) throws IOException {
         String storeName = configStore.getEndpoint();
         Date date = new Date();
-        SettingSelector settingSelector = new SettingSelector();
-        if (!label.equals("%00")) {
-            settingSelector.setLabelFilter(label);
-        }
+        SettingSelector settingSelector = new SettingSelector().setLabelFilter(label);
 
         // * for wildcard match
         settingSelector.setKeyFilter(context + "*");
```

Comment:
Can we have this symbol `"*"` in some static variable with some meaningful name saying what means it?